push-notifications-ios/push-notifications-ios/AppDelegate.swift

```diff
@@ -14,11 +14,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
-        self.pushNotifications.registerDeviceToken(deviceToken) {
-            try? self.pushNotifications.subscribe(interest: "hello", completion: {
-                print("Ready to receive notifications!")
-            })
-        }
+        self.pushNotifications.registerDeviceToken(deviceToken)
```

Comment:
I think you can preserve the completion handler to have the `print("Ready to receive notifications")`